### PR TITLE
added the card layout on route /createPlaylist

### PR DESCRIPTION
**Pull request by shivanshBTW**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,17 +3,21 @@ import { Route, Switch } from 'react-router-dom'
...
-      <Route path="/audiorec" component={AudioRecorder} /> */}
-      </Switch>
-    </div>
-  )
+   return (
+      <div className='App'>
+         <Switch>
+            <Route path={ReactPath.homePath} exact component={MainPage}/>
+            <Route path={ReactPath.scribble} component={ScribblePad}/>
+            <Route path={ReactPath.createPlaylist} component={CreatePlaylist}/>
```
Comment (siddhant1) [inline src/App.js]: Rename createPlaylist -> CreateLecture

File: src/lib/ReactPath.js [hunk 2]
```diff
@@ -2,6 +2,9 @@ class ReactPath {
   static scribble = '/scribble'
   static screenRecord = '/screenRecord'
   static webcam = '/webcam'
+  static createPlaylist = '/createPlaylist'
```
Comment (siddhant1) [inline src/lib/ReactPath.js]: rename

**Comment by siddhant1** [comment]
image
Seriously judging you on the color choice 😆

**Comment by shivanshBTW** [comment]
> image
> Seriously judging you on the color choice 😆

I don't understand what's your problem with the LGBTQ+ community regardless of me being a part of it... but if you consider that a laughing matter, u seriously need to change you mindset

**Comment by shivanshBTW** [comment]
> > image
> > Seriously judging you on the color choice 😆
> 
> I don't understand what's your problem with the LGBTQ+ community regardless of me being a part of it or not... but if you consider that a laughing matter, u seriously need to change you mindset

anyways, this is from the color pallete of Instagram

**Pull request closed without merging**